examples/oneapi/cpp/source/covariance/cov_dense_online.cpp

```diff
@@ -27,7 +27,8 @@ int main(int argc, char const *argv[]) {
     const auto cov_desc = dal::covariance::descriptor{}.set_result_options(
         dal::covariance::result_options::cov_matrix | dal::covariance::result_options::means);
 
-    auto partial_result = dal::covariance::partial_compute_result();
+    dal::covariance::partial_compute_result<> partial_result;
+
     auto input_table = split_table_by_rows<double>(input, 10);
     for (int i = 0; i < 10; i++) {
```

Comment:
what about creating a variable for the number of chunks?

Reply:
Great Idea, will do